### PR TITLE
fix some yamllints with generated jobs

### PR DESCRIPTION
**Pull request by BenTheElder**

### CODE REVIEW

File: experiment/generate_tests.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def write_prow_configs_file(output_file, job_defs):
     with open(output_file, 'w') as fp:
         yaml.dump(
             job_defs, fp, Dumper=yaml.RoundTripDumper, width=float("inf"))
-        fp.write('\n')
+        #fp.write('\n')
```
Comment (fejta) [inline experiment/generate_tests.py]: What is this?

**Comment by BenTheElder** [comment]
> I thought we autobump these now?

we do autobump the images existing, (this didn't involve building any, just updating the references with the script)

There is also a daily autobump that updates the yaml, but I was hoping to unbreak some CI ASAP.

File: experiment/generate_tests.py (hunk 1 above)
Comment (BenTheElder) [reply experiment/generate_tests.py]: will fixup to actually remove this line.

I don't know why we decided to do this before, but the yamllint  presubmit doesn't allow any blank lines so I needed to remove it :/

Comment (BenTheElder) [reply experiment/generate_tests.py]: removed

**Pull request merged**